### ISSUE TITLE
Annotation are not autoloaded in a CLI command

### ISSUE THREAD

**Issue by Jean85** [schmittjoh/JMSSerializerBundle#674]
We're using the bundle to serialize stuff in an API Symfony app. If we want to serialize our objects during a normal request, all is well. If we try to do that inside a console command, we get this error:

> The annotation "@jms\Serializer\Annotation\Type" in property App\Product::$sellersku does not exist, or could not be auto-loaded.'

The specific annotation doesn't matter, the base issue is that annotation are not autoloaded. This stems from https://github.com/schmittjoh/serializer/issues/855#issuecomment-357955279, where @goetas suggests to register the annotation autoloader manually.

Basically, it seems that, during a request, something else is registering the the annotations for autoloading, and that doesn't happen in CLI.

This should be also solved by https://github.com/doctrine/annotations/pull/205, but it will be released with `doctrine/annotations` 2.0, so it's long in the future...

**Comment by Jean85** [comment]
I would like to add a second thing: the annotations are also not loaded in the case that the request doesn't hit a controller, i.e. with a 403. This is because, when hitting a controller, Symfony itself calls `AnnotationRegistry::registerLoader('class_exists')` to load its annotations.

**Comment by Jean85** [comment]
I've talked with @goetas and I've understood that the root of the problem is in the fact that I do not get the serializer through the bundle, but passing through a different route, since I use the `willdurand/hateoas-bundle` and I need to customize a few related things.

In this way, I'm not able to leverage the Symfony annotation loader that is used in this bundle.
